Source/WebKit/UIProcess/WebAuthentication/WebAuthenticatorCoordinatorProxy.cpp

```diff
@@ -79,6 +79,10 @@ void WebAuthenticatorCoordinatorProxy::handleRequest(WebAuthenticationRequestDat
         if (result) {
 #if HAVE(UNIFIED_ASC_AUTH_UI)
             if (!authenticatorManager.isMock() && !authenticatorManager.isVirtual()) {
+                if (!ascSuccessfullySoftlinked()) {
```

Comment:
You should be able to use `isAuthenticationServicesCoreFrameworkAvailable()`, which is generated by the soft linking macros, instead of adding a new method.